### PR TITLE
Scaffolding rails

### PR DESCRIPTION
**Pull request by koggs**
### What we did:

- Scaffold rails application
- Clean commented-out code

### CODE REVIEW

**Comment by GeorgeJLewis** [comment]
Where is the PT link?

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ruby '2.4.1'
+
+gem 'rails', '~> 5.2.0'
+gem 'pg', '>= 0.18', '< 2.0'
+gem 'puma', '~> 3.11'
+gem 'sass-rails', '~> 5.0'
+gem 'uglifier', '>= 1.3.0'
+gem 'coffee-rails', '~> 4.2'
+gem 'turbolinks', '~> 5'
+gem 'jbuilder', '~> 2.5'
```
Comment (tochman) [inline Gemfile]: Please remove this gem

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.4.1'
+
+gem 'rails', '~> 5.2.0'
+gem 'pg', '>= 0.18', '< 2.0'
+gem 'puma', '~> 3.11'
+gem 'sass-rails', '~> 5.0'
+gem 'uglifier', '>= 1.3.0'
+gem 'coffee-rails', '~> 4.2'
```
Comment (tochman) [inline Gemfile]: Please remove this gem

File: Gemfile [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+group :development do
+  gem 'web-console', '>= 3.3.0'
+  gem 'listen', '>= 3.0.5', '< 3.2'
+  gem 'spring'
+  gem 'spring-watcher-listen', '~> 2.0.0'
+end
+
+
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```
Comment (tochman) [inline Gemfile]: Please remove this gem

File: Gemfile [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+gem 'puma', '~> 3.11'
+gem 'sass-rails', '~> 5.0'
+gem 'uglifier', '>= 1.3.0'
+gem 'coffee-rails', '~> 4.2'
+gem 'turbolinks', '~> 5'
+gem 'jbuilder', '~> 2.5'
+gem 'bootsnap', '>= 1.1.0', require: false
+
+group :development, :test do
+  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
```
Comment (tochman) [inline Gemfile]: Please remove this gem

File: config/database.yml [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+# You can use this database configuration with:
+#
+#   production:
+#     url: <%= ENV['DATABASE_URL'] %>
+#
+production:
+  <<: *default
+  database: newsroom-april_production
+  username: newsroom-april
+  password: <%= ENV['NEWSROOM-APRIL_DATABASE_PASSWORD'] %>
```
Comment (tochman) [inline config/database.yml:18]: Please clean up this file from comments

**Review by tochman — CHANGES_REQUESTED** [review]
Please see comments

File: config/database.yml (hunk 5 above)
Comment (koggs) [reply config/database.yml:18]: All comments have been addessed

**Review by tochman — APPROVED** [review]
Good job

**Comment by alghazu** [comment]
Merge it babe

**Pull request merged**